portal/messaging/secure_chat.php

```diff
@@ -1025,7 +1025,7 @@ public function pingAction()
                         <div class="panel-footer box-footer-hide">
                             <form id='msgfrm' ng-submit="saveMessage()">
                                 <div class="input-group">
-                                    <input type="text" placeholder="Type message..." id="msgedit" autofocus="autofocus"
+                                    <input type="text" placeholder=<?php echo xla('Type Message...'); ?> id="msgedit" autofocus="autofocus"
```

Comment:
This needs quote around it (and will break since it has spaces in the constant):
```
<input type="text" placeholder="<?php echo xla('Type Message...'); ?>" id="msgedit" autofocus="autofocus"
```